src/table/hooks/use-scroll-listener.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect } from 'react';
 import { handleHorizontalScroll } from '../utils/handle-scroll';
 
-const useScrollListener = (tableContainerRef: React.MutableRefObject<HTMLDivElement>, direction: string) => {
+const useScrollListener = (tableContainerRef: React.MutableRefObject<HTMLElement>, direction: string) => {
```

Comment:
changing to HTMLElement for consistency

Reply:
There are many
![Screenshot 2022-08-10 at 16 11 05](https://user-images.githubusercontent.com/4471489/183923017-86431265-ce8c-446e-be2f-6c481c0c8254.png)
To be precise, i think it is  HTMLDivElement